tests/queries/0_stateless/02982_enhance_progress_bar.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Test has many issues:
* no-parallel is not necessary
* Creating / deleting a database is not necessary (it's done automatically), just create the table
* `http://localhost:8123` is wrong, it should be `$CLICKHOUSE_URL`. There are many similar examples.

Reply:
Done